Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -337,7 +337,16 @@ extension Generator {
     }
 
     private func makeInlineProperties(for type: TypeName, object: JSONSchema.ObjectContext, context: Context) throws -> [Property] {
-        var keys = object.properties.keys
+        let excludedProperties = excludedProperties(for: type.rawValue)
+        let excludedDiff = excludedProperties.subtracting(object.properties.keys)
+        
+        if !excludedDiff.isEmpty {
+            for diff in excludedDiff {
+                try handle(warning: "Excluded property \(type.rawValue).\(diff) does not exist on schema object \(type.rawValue)")
+            }
+        }
```

Comment:
I know we do it elsewhere, but does calling `for diff in excludedDiff { ... }` when the array is empty actually take a hit in terms of performance? 

I don't know the answer myself but it feels like it would be a no 😄

Reply:
Nope, in fact to be pedantic it adds ~150 instructions for the case where it isn't empty.